src/js/components/Select/Select.js

```diff
@@ -92,28 +96,30 @@ class Select extends Component {
           onClose={this.onClose}
           a11yTitle={`${a11yTitle}${typeof value === 'string' ? `, ${value}` : ''}`}
           dropContent={<SelectContainer {...this.props} onChange={onSelectChange} />}
+          fill={true}
```

Comment:
Why are we adding `fill`?

Reply:
I think the button inside select should fill the available space for the container. right?